test/unit/error.test.ts

```diff
@@ -546,6 +548,56 @@ describe('MongoErrors', () => {
       expect(isResumableError(new MongoNetworkError('ah!'))).to.be.true;
     });
 
+    it('should return false for Error with code', () => {
+      const errorWithCode = new (class extends Error {
+        code = MONGODB_ERROR_CODES.CursorNotFound;
+      })();
+      expect(isResumableError(errorWithCode)).to.be.false;
+    });
+
+    it('should return false nullish arguments', () => {
+      expect(isResumableError()).to.be.false;
+      expect(isResumableError(null)).to.be.false;
+      expect(isResumableError(null, null)).to.be.false;
+    });
+
+    it('should return true for labelless MongoError with CursorNotFound code', () => {
+      const mongoError = new MongoError('ah!');
+      mongoError.code = MONGODB_ERROR_CODES.CursorNotFound;
+      expect(isResumableError(mongoError)).to.be.true;
+      expect(isResumableError(mongoError, 9)).to.be.true;
+      expect(isResumableError(mongoError, 8)).to.be.true;
+    });
+
+    it('should return true for labeled MongoError above wireVersion 9', () => {
+      const mongoError = new MongoError('ah!');
+      mongoError.addErrorLabel(MongoErrorLabel.ResumableChangeStreamError);
+      expect(isResumableError(mongoError)).to.be.false;
+      expect(isResumableError(mongoError, 9)).to.be.true;
+      expect(isResumableError(mongoError, 8)).to.be.false;
+    });
+
+    it('should return false for resumable codes if wireVersion is above 9', () => {
```

Comment:
use "above" for `>`, and "at least" for `>=`
```suggestion
    it('should return false for resumable codes if wireVersion is at least 9', () => {
```